### ISSUE TITLE
Staging Review finding: H1 heading is keyboard focusable when tabbing through page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101263]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Direct Upload
**Feature:** Document Upload

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/introduction](https://staging.va.gov/form-upload/21-0779/introduction), the H1 "Upload VA Form 21-0779" receives keyboard focus when tabbing through the page.

Keyboard users don't typically expect non-interactive elements to be a tab stop, and when they are a tab stop it may prompt them to explore possible interactions (does it do anything if I press enter? etc). That's distracting from the purpose of the page, and inconsistent with other heading behavior on VA.gov. It's appropriate for static elements to receive focus when it's set programmatically (eg. to send someone to a specific spot on the page after an action), but it doesn't really make sense for it to be something you can reach on your own.

<details><summary>Link, screenshot or steps to recreate</summary>

Currently coded as:
```
<h1 tabindex="0">Upload VA Form 21-0779</h1>
```

</details>

## Recommended action

Remove `tabindex="0"` from the H1.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
This has been addressed.

<img width="715" alt="Image" src="https://github.com/user-attachments/assets/b112e459-b50a-4ff0-8a2a-48129c87d09c" />
